content/02.main-text.md

```diff
@@ -317,6 +321,34 @@ While Manubot does require technical expertise, the benefits are manyfold.
 Specifically, Manubot demonstrates a system for publishing that is transparent, reproducible, immediate, permissionless, versioned, automated, collaborative, open, linked, provenanced, decentralized, hackable, interactive, annotated, and free of charge.
 These attributes empower integrating Manubot with an ecosystem of other community-driven tools to make science as open and collaborative as possible.
 
+## Future Manubot Enhancements
+
+Manubot is still under active development, and we envision major changes in its design and dependencies going forward.
+Currently, manuscript repositories must contain a large number of files that do not directly contain manuscript content.
+While this enables a high-degree of customization, it also increases complexity.
+Therefore, we are investigating whether configuration files with sensible defaults could enable bare-bones repositories that contain manuscript content and little else.
+
+In addition, to simpler usage, we're also looking into simplified setup.
+Presently, setup is complex because users must do advanced command-line operations to clone the `manubot/rootstock` repository and configure Travis CI.
+While we provide detailed instructions, users often struggle replicating the long list of setup commands.
```

Comment:
Should we add "... in an appropriate computational environment"?  I'm thinking about the Travis commands and Windows non-compatibility.